test/policy.json

```diff
@@ -10,12 +10,6 @@
                 {
                     "type": "reject"
                 }
-            ],
```

Comment:
This isn't going to work. First, the "show trust" test is going to fail because it explicitly needs this line. Second, you're not removing the `install` command from lib.sh, so all `ubi` pulls are going to fail. Third, the `in-podman` images still have the bad `policy.json` so those are going to fail anyway.

Sorry!

Reply:
Why do we need remove the install command?

Reply:
Oh, good point, I guess with this change you wouldn't need to. Points (1) and (3) still remain though.